tests/testthat/test-container.R

```diff
@@ -45,3 +45,8 @@ test_that("It throws an error when trying to set the name of a container", {
   container <- getContainer(liverPath, sim)
   expect_that(container$name <- "name", throws_error())
 })
+
+test_that("It can print container", {
+  container <- getContainer(liverPath, sim)
+  expect_error(container$print(), NA)

```

Comment:
Expect error?? This reads weird. Can you explain?

Reply:
`expect_error(someCall, NA)` is the way to test for no error being generated in testthat.

Reply:
Ok weird